installer_scripts/install-yb-voyager

```diff
@@ -832,6 +835,25 @@ create_gather_assessment_metadata_dir() {
 	fi
 }
 
+create_guardrail_scripts_dir() {
```

Comment:
also add logic to `rebuild_voyager_local` function so that the scripts get placed in the right location.

Reply:
Added